src/main/java/com/thealgorithms/bitmanipulation/GetBit.java

```diff
@@ -0,0 +1,10 @@
+package com.thealgorithms.bitmanipulation;
+/**
+ * Returns the value of bit from num located at get
+ */
+
+public class GetBit {

```

Comment:
This is a very good start for the [`BitArray` class](https://github.com/TheAlgorithms/Java/issues/4350#issuecomment-1710635231).

Reply:
I suggest adding this as-is and consolidating algorithms into a data structure in a following PR. There are already some other functions implemented in this manner and, as you have mentioned, there are open questions about the design of this BitArray.
@lukasb1b will you be able to consolidate these bit functions into a BitArray in the next PR? If no, somebody else could pick up this task.
In the meantime, we can discuss what we want to use as an underlying type.